backend/LexCore/Entities/Project.cs

```diff
@@ -14,7 +14,17 @@ public class Project : EntityBase
 
     public async Task<Changeset[]> GetChangesets(IHgService hgService)
     {
-        return await hgService.GetChangesets(Code);
+        var age = DateTimeOffset.UtcNow.Subtract(CreatedDate);
```

Comment:
I just realized that it's possible in gql that CreatedDate isn't pulled out of the DB so the value could be the default value. I think there's a way to override this so that a field is always selected from the DB even if it's not selected in the gql query.

Reply:
Nice catch! Indeed, if I didn't select the creation-date in the project-page, the Sena 3 project suddenly had no history 😆.